### PR TITLE
Add bin full alert to HaclCodes

### PR DESCRIPTION
**Pull request by AdamCLarsen**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 setuptools.setup(
     name='frigidaire',
-    version='0.14',
+    version='0.15',
```
Comment (bm1549) [inline setup.py:9]: Can you rebase this on main?

**Review by bm1549 — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (AdamCLarsen) [reply setup.py]: Done.

**Review by bm1549 — APPROVED** [review]

**Comment by bm1549** [comment]
I can't tag the release for the next few days because I'm away and can't do it on mobile but I'll do it when I get back

**Comment by gazpachoking** [comment]
> I can't tag the release for the next few days because I'm away and can't do it on mobile but I'll do it when I get back

I can take care of this one too if you want.

**Comment by bm1549** [comment]
@gazpachoking yes please 🙏

**Comment by gazpachoking** [comment]
0.16 is good to go!

**Pull request merged**